src/components/Skeleton/AudioTrackSkeleton.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <article class="flex flex-row w-full mb-14">
+    <!-- Thumbnail skeleton -->
+    <div
+      class="flex-shrink-0 h-20 w-20 mr-4 bg-dark-charcoal-10 rounded shimmering"
+    >
+      &nbsp;
```

Comment:
Is the space necessary to accomplish this? I tested this in VoiceOver with Safari and while the element is not tabbable of course, if you use arrow key navigation (which is common for reading through paragraphs) VoiceOver will read "space" for each of the skeleton elements becasue of this single space.

Reply:
Thanks for reminding me about this, @sarayourfriend. I used the non-breaking space entity to avoid lint errors about empty elements, but later I changed to auto closed `div`s for fewer characters, but non-void void HTML elements –as divs– technically are not valid in HTML5 (tested with [W3C HTML validator](https://validator.w3.org/), just in XHTML syntax so I was wondering if this could be a bad practice 🤔

I deleted the entities but an alternative could be to disable that rule in these files.

Reply:
I also wondered if this is something Vue does under the hood or just the browser autocompleting tags, as this example fails the W3C validator test, but opening in a browser shows the closing tag.

```html
<!DOCTYPE xhtml>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>
    <div class="myvoiddiv" />
</body>
</html>
```
Anyway, it's just a curiosity that I had. As you said this way works so I'll merge it. Thank you!